visualization/app/codeCharta/ui/attributeTypeSelector/attributeTypeSelector.component.html

```diff
@@ -5,8 +5,8 @@
 		</md-button>
 	</div>
 	<md-menu-content width="2">
-		<md-menu-item style="min-height: 20px; height: 20px">
-			<md-button style="min-height: 20px; height: 20px; line-height: 20px; font-size: 9pt; text-align: center" ng-disabled="true"
+		<md-menu-item style="min-height: 20px; height: 20px;">
+			<md-button style="min-height: 20px; height: 20px; line-height: 20px; font-size: 9pt; text-align: center;" ng-disabled="true"
```

Comment:
How did this sneak in? Is this part of a new linting rule? Prettier didn't add a semicolon before.

Reply:
I just ran prettier against all files *shrug*

Reply:
As long as prettier keeps this format I'm fine. :)

Reply:
I just run `npm run format` and it removed those semicolons. I'll commit my formatation.